extensions/amp-subscriptions/0.1/local-subscription-platform.js

```diff
@@ -177,14 +183,10 @@ export class LocalSubscriptionPlatform {
         })
         .then(res => res.json())
         .then(resJson => {
-          const entitlements = new Entitlements(
-              this.serviceConfig_['serviceId'] || 'local',
-              JSON.stringify(resJson),
-              Entitlement.parseListFromJson(resJson),
-              currentProductId
-          );
-          this.entitlements_ = entitlements;
-          return entitlements;
+          const entitlement = Entitlement.parseFromJson(resJson);
+          entitlement.setCurrentProduct(currentProductId);
```

Comment:
`setCurrentProduct` should be called somewhere in `EntitlementsStore`. I'd also recommend to do the same for "service" property.

Reply:
My reaction here is that this would require every platform to call `setCurrentProduct` even though the central `amp-subscriptions` code can do it for them since it knows all about the current product given that it's a document-central thing.

Reply:
Done